dockers/docker-orchagent/start.sh

```diff
@@ -12,6 +12,17 @@ function start_app {
     done
 }
 
+function config_acl {
+    if [ -f "/etc/sonic/acl.json" ]; then
+        mkdir -p /etc/swss/config.d/acl
+        rm -rf /etc/swss/config.d/acl/*
```

Comment:
We currently don't have a config.sh for swss docker and I don't feel it's really necessary to add one just for acl.